jira/client.py

```diff
@@ -2813,29 +2813,23 @@ def delete_user_avatar(self, username, avatar):
         return self._session.delete(url, params=params)
 
     def search_users(
-        self, user, startAt=0, maxResults=50, includeActive=True, includeInactive=False
+        self, query, startAt=0, maxResults=50
     ):
         """Get a list of user Resources that match the specified search string.
 
-        :param user: a string to match usernames, name or email against.
-        :type user: str
+        :param query: a string to match usernames, name or email against.
```

Comment:
Fundamentally the function is still asking for a 'user', it is just the API that puts this under the 'query' request parameter. Therefore, I think keeping the argument as 'user' is fine